sig-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# SIG: Ops
+
+This Special Interest Group (SIG) main goal is to coordinate Operations (Ops) activities and best practices across Spinnaker Operators.
+* Production Readiness
```

Comment:
Would it make sense to explicitly include monitoring Spinnaker under the purview of this SIG?

Reply:
+1, monitoring is one of the most asked-about but currently SIG-less topics.

Reply:
+1

Reply:
Yes, Monitoring and logging are critical areas of Ops

Reply:
I think it makes sense to have include monitoring. Some topics we could cover:
- **Observability** => centralize logging, infrastructure and platform monitoring, availability and performance dashboards
- **Disaster recovery** => recovery of pipelines, applications or running executions. 
- **Replication** => Multi spinnaker landscapes

Reply:
Yes, I think Auditing could possibly be grouped in there as well

Reply:
How about scaling?

Reply:
added all those!